Add alert and runbook when tenant mapping fails (#67)
appuio/component-appuio-cloud-reporting: docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingDatabaseTenantMappingFailed.adoc

= Alert rule: APPUiOCloudReportingDatabaseTenantMappingFailed

include::partial$runbooks/contribution_note.adoc[]

== icon:glasses[] Overview

APPUiO Cloud Reporting failed while trying to add a tenant target mapping from billing metrics to the reporting database.

> This can lead to problems with the invoice generation but shouldn't be time critical.

Reviewer: nb: Some additional information on when this is time critical might be helpful.

```suggestion
This can lead to problems during the monthly invoice generation.
```
